storage: remove spurious debugging logs

tbg opened the pull request:
These accidentally slipped in in
adc7b88ebfdc1dcdbc89f5cdbb4f709af06d7726.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/31871)
<!-- Reviewable:end -->

tbg committed:
storage: remove spurious debugging logs

These accidentally slipped in in
adc7b88ebfdc1dcdbc89f5cdbb4f709af06d7726.

Release note: None

tschottdorf commented:
Oops, forgot to delete the line with the args to `Infof`.

bors r=petermattis

ridwanmsharif reviewed (approved):
My bad, thanks for cleaning that up!
